onnxruntime/contrib_ops/rocm/math/bias_softmax_impl.cu

```diff
@@ -0,0 +1,338 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "contrib_ops/rocm/math/bias_softmax.h"
+
+#include <limits>
+#include <algorithm>
+
+
+#include "hip/hip_runtime.h"
+#include "core/providers/rocm/cu_inc/common.cuh"
+#include "core/providers/common.h"
+#include "core/providers/rocm/miopen_common.h"
+#include "core/providers/rocm/shared_inc/accumulation_type.h"
+
+// #include "core/providers/common.h"
+// #include "core/providers/rocm/cuda_common.h"
+// #include "core/providers/rocm/cudnn_common.h"
+#include "core/providers/rocm/cu_inc/binary_elementwise_impl.cuh"
+// #include "core/providers/rocm/cu_inc/common.cuh"
```

Comment:
nit. how about deleting these commented header files?

Reply:
Ah, missed this. Thanks for point it out.